test/functional/tests/lazy_writes/test_lazy_writes_clean.py

```diff
@@ -0,0 +1,343 @@
+#
+from time import sleep
+import pytest
+
+from api.cas import casadm
+from api.cas.cache_config import CacheMode, CleaningPolicy, CacheModeTrait, SeqCutOffPolicy
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from core.test_run import TestRun
+from test_tools.dd import Dd
+from test_tools.disk_utils import Filesystem
+from test_tools.fs_utils import create_random_test_file, remove
+from test_tools.iostat import IOstatBasic
+from test_utils.filesystem.file import File
+from test_utils.os_utils import Udev, sync
+from test_utils.size import Size, Unit
+
+bs = Size(512, Unit.KibiByte)
+mnt_point = "/mnt/cas/"
+
+
+@pytest.mark.parametrize("cache_mode", CacheMode.with_traits(CacheModeTrait.LazyWrites))
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_clean_stop_cache(cache_mode):
+    """
+        title: Test of the ability to stop cache in modes with lazy writes.
+        description: |
+          Test if OpenCAS stops cache in modes with lazy writes without data loss.
+        pass_criteria:
+          - Cache stopping works properly.
+          - Writes to exported object and core device during OpenCAS's work are equal
+          - Data on core device is correct after cache is stopped.
+    """
+    with TestRun.step("Prepare devices for cache and core."):
+        cache_dev = TestRun.disks['cache']
+        cache_dev.create_partitions([Size(256, Unit.MebiByte)])
+        cache_part = cache_dev.partitions[0]
+        core_dev = TestRun.disks['core']
+        core_dev.create_partitions([Size(512, Unit.MebiByte)])
+        core_part = core_dev.partitions[0]
+        Udev.disable()
+
+    with TestRun.step(f"Start cache in {cache_mode} mode."):
+        cache = casadm.start_cache(cache_part, cache_mode)
+
+    with TestRun.step("Add core to cache."):
+        core = cache.add_core(core_part)
+        # get IO stats independent from test - adding core creates metadata on core device
+        core_disk_writes_diff = IOstatBasic.get_iostat_list([core_part])[0].total_writes.value
+
+    with TestRun.step("Disable cleaning and sequential cutoff policies."):
+        cache.set_cleaning_policy(CleaningPolicy.nop)
+        cache.set_seq_cutoff_policy(SeqCutOffPolicy.never)
+
+    with TestRun.step("Write data to the exported object."):
+        test_file_main = create_random_test_file("/tmp/test_file_main", Size(64, Unit.MebiByte))
+        dd = Dd().output(core.system_path) \
+            .input(test_file_main.full_path) \
+            .block_size(bs) \
+            .count(int(test_file_main.size / bs)) \
+            .oflag("direct")
+        dd.run()
+        test_file_md5sum_main = test_file_main.md5sum()
+
+    with TestRun.step("Check IO stats."):
+        # get IO stats generated by test
+        core_disk_writes = (IOstatBasic.get_iostat_list([core_part])[0].total_writes.value
```

Comment:
So, core_disk_writes is the DIFFERENCE between the current value of writes and the initial value of writes after creating partitions (which you named _diff). Current names do not reflect the actual values of those variables.